<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>测绘基础理论与方法(题2)</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rec.qrc">
    <normaloff>:/icon/linear.ico</normaloff>:/icon/linear.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>数据管理器</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="5" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>拟合结果：等待计算...</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QRadioButton" name="radioButton_x">
             <property name="text">
              <string>Y 是观测值（X 无误差）普通最小二乘法（OLS）</string>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_xy1">
             <property name="text">
              <string>XY 都是观测值  (特征值分解法)总体最小二乘法（TLS）</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_xy2">
             <property name="text">
              <string>XY 都是观测值  (奇异值分解法)总体最小二乘法（TLS）</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>计算平差</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="pushButton_daoru">
             <property name="text">
              <string>导入数据</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_tianjia">
             <property name="text">
              <string>添加点</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_shanchu">
             <property name="text">
              <string>删除选中点</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QTableWidget" name="tableWidget">
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>X</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Y</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>导出图片(PNG)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>图形显示</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="GraphicsView" name="graphicsView"/>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>缩放控制：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_fangda">
             <property name="text">
              <string>放大</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_suoxiao">
             <property name="text">
              <string>缩小</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_chongzhi">
             <property name="text">
              <string>重置</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>715</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">graphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="rec.qrc"/>
 </resources>
 <connections/>
</ui>
